client/src/pages/Login.vue

```diff
@@ -28,7 +25,7 @@
             v-model="form.password"
             :type="isPwd ? 'password' : 'text'"
             :label="$t('auth.login_fields.password')"
-            @keypress.enter="onSubmit"
+            @keypress.enter="login"
```

Comment:
Add autocomplete="current-password" attribute.

Ref: https://web.dev/sign-in-form-best-practices/

Reply:
I've added this missing autocomplete attribute to the password input.